.github/workflows/CompatHelper.yml

```
@@ -0,0 +1,19 @@
+name: CompatHelper
+
+on:
+  schedule:
+    - cron: '00 00 * * *'
+
+jobs:
+  CompatHelper:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.3
```

Maybe use version 1.4?
```suggestion
          version: 1.4
```